commitlint/plugins.ts

```diff
@@ -369,18 +431,13 @@ export abstract class Plugins {
     }
 
     public static bodySoftMaxLineLength(
-        rawStr: string,
+        bodyStr: string | null,
```

Comment:
@realmarv I understand that in this commit you need to change the rules that receive bodyStr because there might be no body (so, null); however, the other rules that don't deal with the body should not change, in those cases you can assert that rawStr is not null and they receive string, not `string | null`.

Reply:
The output of the convertAnyToString function can be null or string and its output is the input to the plugins. What do you suggest me to do?

Reply:
in the cases where you call convertAnyToString with raw, cast the result from `string|null` to a `string` (not sure if this is done via an assert)

Reply:
I'm not sure an assert works but I can use casting like this `(Helpers.convertAnyToString(raw, "raw") as string)`. What do you think?

Reply:
This is still not resolved. By the way, I checked the log of the last CI run and it is using 17.4.3 so you should rename this PR to not specify 17.4.3, maybe just say "latest".

Reply:
typo in above msg ^ , edited it